src/NServiceBus.Gateway.AcceptanceTests/GatewayTestSuiteConstraints.cs

```diff
@@ -2,6 +2,5 @@
 {
     public partial class GatewayTestSuiteConstraints : IGatewayTestSuiteConstraints
     {
-        public static GatewayTestSuiteConstraints Current = new GatewayTestSuiteConstraints();
```

Comment:
Will this break the downstreams?

Reply:
In core it gives access to special properties about whether the configuration supports certain features or not, but that doesn't seem to be the case with the gateway